tree/ntupleutil/v7/src/RNTupleInspector.cxx

```diff
@@ -181,6 +181,24 @@ ROOT::Experimental::RNTupleInspector::Create(std::string_view ntupleName, std::s
    return inspector;
 }
 
+std::string ROOT::Experimental::RNTupleInspector::GetCompressionSettingsAsString() const
+{
+   int algorithm = fCompressionSettings / 100;
+   int level = fCompressionSettings - (algorithm * 100);
+
+   std::string algorithmName;
+   switch (algorithm) {
+   case 1: algorithmName = "zlib"; break;
+   case 2: algorithmName = "LZMA"; break;
+   case 3: algorithmName = "Old compression algorithm"; break;
+   case 4: algorithmName = "lz4"; break;
+   case 5: algorithmName = "zstd"; break;
+   default: algorithmName = "Undefined compression algorithm";
+   }
```

Comment:
This part is not specific to RNTuple and probably belong in core (close to or in Compression.h)

Reply:
Good point, I'll move it there!